nodestream/project/project.py

```diff
@@ -118,7 +118,9 @@ def to_file_data(self) -> dict:
                 for scope in self.scopes_by_name.values()
                 if scope.persist
             },
-            "plugins": [plugin.to_file_data() for plugin in self.plugins or []],
+            "plugins": [
+                plugin.to_file_data() for plugin in self.plugins_by_name.values() or []
```

Comment:
I don't think the `or []` is not required since `.values()` will always be defined. 

```suggestion
                plugin.to_file_data() for plugin in self.plugins_by_name.values()
```